torch/_prims/__init__.py

```diff
@@ -1313,7 +1313,9 @@ def _collapse_view_helper(
     return new_shape, new_strides
 
 
-def _collapse_view_meta(a: TensorLikeType, start: int, end: int) -> TensorLikeType:
+def _collapse_view_meta(
+    a: TensorLikeType, origianl_shape: ShapeType, start: int, end: int
```

Comment:
can't original shape be inferred from `a`?

Reply:
https://github.com/pytorch/pytorch/pull/83739/files#r950352124
I'll try to find some other way.